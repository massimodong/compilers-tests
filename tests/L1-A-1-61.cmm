struct zIVpj6 {
  float CknvV6aw , SddQc , orBKkyLon7r 415645814.1141600253 
    ;
  int zbe0weTOiBFRy8XFk7 [ 286717468 ] [ 619414664 
    ] [ 1573626770 ] [ 1295236209 != ] , l7Ggj = _3rf 
    , JksscDl ;
  struct {
  }Mf2NHnOGoTgz [ 2028856280 ] = - wFuYgceU4l = RLncz 
    , , M ;
  struct b3im7W  zq [ 2009994871 ] ;
  float bSlEuK = cYn , XoqYFDcj1 ;. 
  int qJIA1WPznYeI5 ;
}bWnMekCE 
;
