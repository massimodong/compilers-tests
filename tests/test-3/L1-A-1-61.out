Error Type B at Line 2
Error Type B at Line 5
Error Type B at Line 9
Error Type B at Line 10
Error Type B at Line 11
